### DISCUSSION TITLE
How can I change the logs to be in English?

### DISCUSSION THREAD

**Discussion by priyam-raj** [Androz2091/AtlantaBot#468] [Q&A] [answered]
Everything else that the Bot does is in English by default except for these logs, any way to change it to be English by default?
![image](https://user-images.githubusercontent.com/26673536/117808802-f6b8ea00-b27a-11eb-9356-548e1bf38f8b.png)

**Comment by karlof002** [comment, marked as answer]
Hey, you have to check those both files => `guildCreate.js` & `guildDelete.js`, search for the veriable called `text` and translate it in english in your case, regardless
